mongodb/migrations/migrations.go

```diff
@@ -154,12 +154,39 @@ var m003 = migrate.Migration{
 	},
 }
 
+var m004 = migrate.Migration{
+	Version:     4,
+	Description: "set empty path field on all ipnskeys",
```

Comment:
could have detected it missing on every lookup, but felt cleaner to do this once and forget about it